### PR TITLE
Changed number of car seats to 5

### LINKED ISSUE #9565: Change number of car seats to 5
#### Challenge Name

https://www.freecodecamp.com/challenges/make-instances-of-objects-with-a-constructor-function

#### Issue Description

Same issue as #9544 but in another challenge.

#### To Fix
[This line in an example block](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/a9c172454fcc1b637830a5fd6ff3cdf20d0543b6/seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json#L120), [this line in the challenge seed](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/a9c172454fcc1b637830a5fd6ff3cdf20d0543b6/seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json#L132), and [this line in the solution](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/a9c172454fcc1b637830a5fd6ff3cdf20d0543b6/seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json#L143) needs to be changed to have 5 seats (e.g. `this.seats = 5;`).

#### To Contributors

Please first read the [contributing guidelines](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before taking care of this issue. And feel free to visit the [Contributors Help](https://gitter.im/FreeCodeCamp/HelpContributors) chat room if you have any questions about helping. We're there to help. 

This issue is labeled `first-timers-only` so only those who have not contributed to Free Code Camp's main repository may resolve this issue. 

#### Screenshot

image

### PR DESCRIPTION
**Pull request by ladybugtju**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #9565
#### Description

OOP challenges _Car Seats number correction for both Eng/Sp versions

### CODE REVIEW

**Comment by raisedadead** [comment]
@ladybugtju thanks a lot for the pull request. 

Can you please change the commit message and the PR title above to something more understandable?
> Changed number of car seats to 5

To do so you should:

`git checkout fix/car_seats`
`git commit --amend`
Edit the commit message to `Changed number of car seats to 5`
`git push origin fix/car_seats --force`

Then change the pull request title on this page with the edit option.

Let us know if you need any help in the [Contributors Chat room](https://gitter.im/FreeCodeCamp/HelpContributors).

Also here is a good read if you have some time:
http://forum.freecodecamp.com/t/writing-good-git-commit-messages/13210

**Comment by ladybugtju** [comment]
I think the title is quite clear Object Oriented Porgramming: Change car seats to 5 in English and Spanish. I tried to keep it short. I could change it to something else if required. Please mention what and why

**Comment by ladybugtju** [comment]
@raisedadead Hi there, I m not that convinced yet but wanted to play with commit message changes. Hope this works. Thanks for the info. Keep me posted

**Comment by raisedadead** [comment]
@ladybugtju for sure, this being a community project we need to have commit messages that can be understood down the lane, even when existing contributors are not there any longer.

Its not an enforcement but an advisement. 

As I mentioned earlier:
> Also here is a good read if you have some time:
> http://forum.freecodecamp.com/t/writing-good-git-commit-messages/13210

nevertheless have a look at the below article and you will get to know what we are trying to look forward to, especially from new contributors:
http://chris.beams.io/posts/git-commit/
http://imgs.xkcd.com/comics/git_commit.png

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 1]
```diff
@@ -188,7 +188,7 @@
         "  //Change this constructor",
         "  this.wheels = 4;",
         "  this.seats = 1;",
-        "  this.engines = 1;",
+        "  this.engines = 5;",
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: This should be changed back to `this.engines = 1` and the line above it should be changed to `this.seats = 5`.

**Comment by erictleung** [comment]
@ladybugtju please change the line I've indicated above. You can make your changes and then just amend your commit message like so:

```sh
cd path/to/object-oriented-and-functional-programming.json # change dir to wherever your file is
git checkout fix/car_seats
... # make changes here 
git add object-oriented-and-functional-programming.json
git commit --amend
git push origin fix/car_seats --force
```

Otherwise, looks good to me 👍 Let us know if you have any questions about squashing or amending commits.

**Comment by bugron** [comment]
@ladybugtju what's your progress on this PR?

**Comment by ladybugtju** [comment]
Hi @bugron, just saw this! Fixed the parts Eric mentioned. Let s see how this goes now. @erictleung: Hi there, thanks for the info. Please check if the last commit is fine

**Comment by bugron** [comment]
@ladybugtju thanks for the update. Waiting for @erictleung to check.

**Comment by erictleung** [comment]
@ladybugtju LGTM! :+1: Thanks for your contribution to FreeCodeCamp! :sparkles: :tada: :sparkles:

**Comment by ladybugtju** [comment]
@erictleung Awesome! Thanks

**Pull request merged**